app/views/diary_entry/view.html.erb

```diff
@@ -21,6 +21,11 @@
     <%= richtext_area :diary_comment, :body, :cols => 80, :rows => 15 %>
     <%= submit_tag t('diary_entry.view.save_button') %>
   <% end %>
+  <% if @user and @entry.subscribers.exists?(@user.id) %>
```

Comment:
I don't see why. I can imagine a situation where a post attracts a lot of trolls and you just want to ignore it and move on for example?